games/o.yaml

```diff
@@ -1644,6 +1644,24 @@
   type: remake
   updated: 2019-05-29
 
+- name: OpenNox
+  originals:
+    - Nox
+  type: remake
+  repo: https://github.com/noxworld-dev/opennox
+  url: https://www.patreon.com/opennox
+  development: active
+  status: playable
+  multiplayer:
+    - Online
+    - LAN
+  content: commercial
+  lang:
+    - Go
+  license:
+    - GPL3
+  updated: 2022-04-11
```

Comment:
```suggestion
  updated: 2022-04-11
  video:
    youtube: K7yqrxeZQVA
```

Reply:
Thank you for adding the YouTube link! It's not made by our team, but still illustrates it pretty well, so adding it.